README.md

```diff
@@ -38,5 +39,4 @@ $ open http://localhost:9000
 
 ### IntelliJ
 
-1. `$ sbt gen-idea`
-2. Inside IntelliJ, `File/Open...`, choose the root folder to import all the projects (do *not* use `Import Project...` or `Import Module...`)
+IntelliJ integration does not work well with cross projects for now. The IDEA team is working on a fix.
```

Comment:
Have you noticed it working better now that https://youtrack.jetbrains.com/issue/SCL-7859 has been fixed? Or is it something else?